### PR TITLE
Writing efficient numerical code in D

### PR DESCRIPTION
**Pull request by ljubobratovicrelja**
I've changed release date to Saturday, the 17th, but we'll change it to whatever if needed.

Ping @9il @wilzbach

### CODE REVIEW

File: _posts/2016-12-17-writing-efficient-numerical-code.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+... This code can be rewritten like so:
+
+```d
+import mir.ndslice.algorithm : ndEach;
+
+image.ndEach!( (ref v)
+{
+    /* perform that processing from inside those loops */
+}, Yes.vectorized, Yes.fastmath );
```
Comment (9il) [inline _posts/2016-12-17-writing-efficient-numerical-code.md]: `Yes.fastmath` is useless for lambdas. A function with @fasmath should be defined.

Comment (ljubobratovicrelja) [reply _posts/2016-12-17-writing-efficient-numerical-code.md]: Ah, tnx! This is second time you're telling me this, sorry for that...

**Comment by 9il** [comment]
LGTM, feel free to merge

**Comment by 9il** [comment]
We will be able to add hot fixes anyway

**Comment by ljubobratovicrelja** [comment]
> We will be able to add hot fixes anyway

Of course! Should we change the release date so it's shown on blog right away?

**Comment by 9il** [comment]
> Should we change the release date so it's shown on blog right away?

Yep, you can change it

**Comment by 9il** [comment]
Please fix link form your mir fork to libmir

**Comment by 9il** [comment]
You can just fix the master. It is OK

**Comment by ljubobratovicrelja** [comment]
> Please fix link form your mir fork to libmir

Fixed 468a97a9092048be69afffb1b06dea1780bce4a4. Sorry about that.

**Pull request merged**